docs/attributes-registry/tls.md

```diff
@@ -0,0 +1,49 @@
+<!--- Hugo front matter used to generate the website version of this page:
+--->
+
+# TLS
+
+## TLS Attributes
+
+<!-- semconv registry.tls(omit_requirement_level) -->
```

Comment:
I think we could skip `omit_requirement_level` as there is none in registry.

Also could you add this file to the readme.md inside attributes-registry?

Reply:
`omit_requirement_level` is contained in all the other .md files in attributes-registry as well. My assumption was that this is an instruction to the table generator?

I'll add the entry to readme.md

Reply:
yes, it's instruction not to render it in the table
you are right we need to have it because it's generic one, my bad